src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -94,6 +95,13 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
     return html`
       ${this.renderStyle()}
       <ha-card>
+        ${stateObj.state === "unavailable"
+          ? html`
+              <div id="overlay" @click="${this._stopPropagation}">
```

Comment:
It's better to not use ids but use classes. So call this one the `.disabled-overlay` and target the text with `.disabled-overlay div`